migrations/versions/c27fec1bfee2.py

```diff
@@ -0,0 +1,35 @@
+"""empty message
```

Comment:
There is already a migration for this in the repo, right?

Reply:
https://github.com/canonical/ubuntu-com-security-api/blob/main/migrations/versions/c27fec1bfee2_.py

Reply:
Yes, it's the same one actually